apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -419,6 +419,29 @@
                     },
                     "minItems": 1,
                     "default" : ["ak_2KAcA2Pp1nrR8Wkt3FtCkReGzAi8vJ9Snxa4PcmrthVx8AhPe8:109"]
+                },
+                "garbage_collection" : {
+                    "type" : "object",
+                    "additionalProperties" : false,
+                    "properties" : {
+                        "enabled": {
+                            "description": "If true, node will perform garbage collection of account state, removing unrechable merkle patricia nodes and free some disk space",
+                            "type": "boolean",
+                            "default": true
+                        },
+                        "interval" : {
+                            "description": "How often (every `interval` block) should garbage collection run",
+                            "type": "integer",
+                            "minimum": 50,
```

Comment:
I'd expect this to be allowed as low as one wants? I tested with a value of 2 which worked fine.

Reply:
Lowered to 5, for the adventurous :)